external-crates/move/crates/move-vm-runtime/src/debug.rs

```diff
@@ -107,22 +128,20 @@ impl DebugContext {
         interp: &Interpreter,
     ) {
         let instr_string = format!("{:?}", instr);
-        let function_string = function_desc.pretty_string();
-        let breakpoint_hit = self.breakpoints.contains(&function_string)
-            || self
-                .breakpoints
-                .iter()
-                .any(|bp| instr_string[..].starts_with(bp.as_str()));
+        let function_string = function_desc.pretty_short_string();
+        let breakpoint_hit = self
+            .breakpoints
+            .get(&function_string)
+            .or_else(|| {
+                self.breakpoints
+                    .iter()
+                    .find(|bp| instr_string[..].starts_with(bp.as_str()))
+            })
+            .or_else(|| self.breakpoints.get(&pc.to_string()));
 
-        if self.should_take_input || breakpoint_hit {
+        if self.should_take_input || breakpoint_hit.is_some() {
             self.should_take_input = true;
-            if breakpoint_hit {
-                let bp_match = self
-                    .breakpoints
-                    .iter()
-                    .find(|bp| instr_string.starts_with(bp.as_str()))
-                    .unwrap()
-                    .clone();
+            if let Some(bp_match) = breakpoint_hit {
                 println!(
                     "Breakpoint {} hit with instruction {}",
                     bp_match, instr_string
```

Comment:
I had trouble with this specific line here where the instr_string was crashing when I ran locally. 

I believe what I input was `breakpoint package::module::function`

Reply:
That should be fixed now!